app/pandas.ipynb

```diff
@@ -515,8 +841,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
+   "execution_count": 95,
+   "outputs": [
+    {
+     "ename": "TypeError",
+     "evalue": "'Series' object is not callable",
+     "output_type": "error",
+     "traceback": [
+      "\u001B[1;31m---------------------------------------------------------------------------\u001B[0m",
+      "\u001B[1;31mTypeError\u001B[0m                                 Traceback (most recent call last)",
+      "File \u001B[1;32m<timed eval>:1\u001B[0m\n",
+      "Cell \u001B[1;32mIn[95], line 9\u001B[0m, in \u001B[0;36munknown_signature\u001B[1;34m()\u001B[0m\n\u001B[0;32m      7\u001B[0m \u001B[38;5;66;03m# Calculate the moving average with a window of 5\u001B[39;00m\n\u001B[0;32m      8\u001B[0m moving_averages \u001B[38;5;241m=\u001B[39m []\n\u001B[1;32m----> 9\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m i \u001B[38;5;129;01min\u001B[39;00m \u001B[38;5;28mrange\u001B[39m(\u001B[38;5;241m4\u001B[39m, \u001B[38;5;28;43mlen\u001B[39;49m\u001B[43m(\u001B[49m\u001B[43mdata\u001B[49m\u001B[43m)\u001B[49m):\n\u001B[0;32m     10\u001B[0m     window \u001B[38;5;241m=\u001B[39m data[i \u001B[38;5;241m-\u001B[39m \u001B[38;5;241m4\u001B[39m : i \u001B[38;5;241m+\u001B[39m \u001B[38;5;241m1\u001B[39m]\n\u001B[0;32m     11\u001B[0m     average \u001B[38;5;241m=\u001B[39m \u001B[38;5;28msum\u001B[39m(window) \u001B[38;5;241m/\u001B[39m \u001B[38;5;241m5\u001B[39m\n",
+      "\u001B[1;31mTypeError\u001B[0m: 'Series' object is not callable"
+     ]
+    }
+   ],
```

Comment:
whoops, something is wrong

Reply:
![image](https://github.com/mate-academy/py-pandas/assets/155154439/ad75a43d-7dbc-4bea-83fd-c6191b4a505a)

its optional task.